Makefile

```diff
@@ -58,6 +60,10 @@ gen:
 		--git-repo-id ${GIT_REPO} \
 		-o /local/${PACKAGE_PREFIX}/${PACKAGE_MAJOR} \
 		-i /local/${SPEC_PATCHED_FILE}
+	# generated tests have a broken import
+	find ${PACKAGE_PREFIX}/${PACKAGE_MAJOR}/test -type f -exec sed -i '' 's_"github.com\/equinix-labs\/metal-go"_"github.com\/equinix-labs\/metal-go\/metal\/v1"_g' {} \;
```

Comment:
I want to experiment with setting `isGoSubmodule=true` to see if it allows us to eliminate this find/replace.